nifi/nifi-nar-bundles/nifi-kite-bundle/pom.xml

```diff
@@ -0,0 +1,60 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.nifi</groupId>
+    <artifactId>nifi-nar-bundles</artifactId>
+    <version>0.0.1-incubating-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>nifi-kite-bundle</artifactId>
+  <version>0.0.1-incubating-SNAPSHOT</version>
```

Comment:
I'm used to just seeing the parent have a version. If NiFi as a community doesn't have any particular rationale for also including version numbers on the component as well (e.g. if the nifi-nar-bundles pom is expected to be released on a different cycle then the included bundles themselves), we should favor just omitting the version on the artifact so that it will come from the parent.